CRM/Case/Form/Activity.php

```diff
@@ -699,14 +699,7 @@ public function postProcess($params = NULL) {
         }
       }
 
-      CRM_Core_Session::setStatus('', ts("'%1' activity has been %2. %3 %4",
-        array(
-          1 => $this->_activityTypeName,
-          2 => $recordStatus,
-          3 => $followupStatus,
-          4 => $mailStatus,
-        )
-      ), 'info');
+      CRM_Core_Session::setStatus('', $recordStatus . ' ' . $followupStatus . ' ' . $mailStatus, 'info');
```

Comment:
Looking at how the message will get formatted, it looks like you're leaving the "body" blank and concatenating all the messages into the "title". (granted, you're not the one who originally did that here)
Might it look better to put the recordStatus in the title and the rest in the body? Like

    CRM_Core_Session::setStatus($followupStatus . ' ' . $mailStatus, $recordStatus, 'success');

It's still a long title though. This is when we're just getting into personal preference rather than anything technical, but I tend to keep messages as simple as possible (for users and translators) and rather than 2 messages (one for create, the other for update) I'd probably just write `ts("%1' Saved.", array(1 => $this->_activityTypeName));` for both titles.

Also this is a success message, so maybe "success" is a better status than "info".